app/components/views/TicketsPage/GovernanceTab/index.js

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import { autobind } from "core-decorators";
 import VotingPrefsPage from "./Page";
-import votingPrefs from "../../connectors/votingPrefs";
-import { find, compose, eq, get, substruct } from "../../fp";
+import votingPrefs from "../../../../connectors/votingPrefs";
+import { find, compose, eq, get, substruct } from "../../../../fp";
 
```

Comment:
#779 obviates the need for this.

You can just do an absolute path from "app" or "components" now.

So lines 4 and 5 can just be
```
import { votingPrefs } from "connectors";
import { find, compose, eq, get, substruct } from "fp";
```